packages/aragon-wrapper/src/index.js

```diff
@@ -311,12 +310,9 @@ export default class Aragon {
       publishReplay(1)
     )
 
-    if (cachedPermissions) {
-      const permissons = of(cachedPermissions).pipe(publish())
-      this.permissions = merge(permissons, fetchedPermissions).pipe(publishReplay(1))
-    } else {
-      this.permissions = fetchedPermissions
-    }
+    const cachedPermissions$ = cachedPermissions ? of(cachedPermissions) : of()
+    this.permissions = concat(cachedPermissions$, fetchedPermissions$).pipe(publishReplay(1))
+    fetchedPermissions$.connect()
```

Comment:
Nitpick: I generally prefer grouping the `.connect()` as close as possible with the `publish()` / `share()` / etc to avoid forgetting it.

There are times where you would want to leave all the `.connect()`s to the end (e.g. if you had a bunch of synchronous observables that should all be subscribed to at the same time), but in this case it's not important as `fetchedPermissions$` is async.

Reply:
Absolutely. In this case the move it quite minimal as it will go up two lines.